recipe/meta.yaml

```diff
@@ -18,25 +18,26 @@ build:
     - image_to_iq = xpdtools.cli.process_tiff:run_main
 
 requirements:
-  host:
+  build:
     - python >=3
-    - pip 
+    - pip
   run:
     - python >=3
-    - fabio
-    - fire
-    - matplotlib-base
-    - numba
     - numpy
-    - pims
-    - pyfai
+    - scipy
+    - scikit-beam
     - pyyaml
+    - pyfai
     - rapidz
-    - scikit-beam
+    - fire
+    - pims
+    - fabio
+    - numba
+    - matplotlib-base
+    - tqdm
     - scikit-learn
-    - scipy
     - tomopy
-    - tqdm
+
```

Comment:
We tend to sort the dependency list alphabetically for easier mainenance. Could you please sort it?

Reply:
Thank you for the suggestion. It's done.